configure

```diff
@@ -110,6 +112,7 @@ sysconfdir ?= ${sysconfdir:-${prefix}/etc}
 sbindir ?= ${sbindir:-${prefix}/sbin}
 mandir ?= ${mandir:-${prefix}/share/man}
 enable_documentation ?= ${enable_documentation:-yes}
+enable_dracut_cpio ?= ${enable_dracut_cpio}
```

Comment:
I'd really appreaciate to add tests for rust and its friends on early stage, rather that bail out later.

Reply:
Fair enough. I'll add a check for cargo presence in the `--enable-dracut-cpio` configure path.